tests/shared/src/test/scala/scala/meta/tests/prettyprinters/PublicSuite.scala

```diff
@@ -219,9 +204,11 @@ class PublicSuite extends TreeSuiteBase {
   }
 
   test("scala.meta.inputs.Position.Range.toString") {
-    val Term.ApplyInfix(lhs, _, _, _) = "foo + bar".parse[Term].get
-    lhs.pos match { case _: Position.Range => ; case _ => }
```

Comment:
this thing wasn't doing anything, was it?

Reply:
I don't think it was, removed.